content/utils.js

```diff
@@ -26,21 +29,21 @@ function memoize(fn) {
     return fn;
   }
 
-  const cache = new Map();
+  const cache = new LRU({ max: 2000 });
```

Comment:
Can we make this a bit bigger? We know that storing 10,000 keys in the `Map` worked fine in GitHub Actions without blowing through any memory limits. 
I noticed that it disposes a lot of keys whereas the old dumb-simple `Map` just did 1 or 2 `.clear()` for building the whole `en-US` tree. 
Not sure if there's an actual performance difference in the grand total though :)

Reply:
I ran it a few times and 2000 seemed fair. Caching behaves different here as some of the larger docs stay in cache so we have to be more defensive. On my machine I got a 10% seed-up vs the old cache. Let's test on CI?